src/components/02_atoms/Widgets/BooleanCheckbox.js

```diff
@@ -5,7 +5,9 @@ import FormControlLabel from '@material-ui/core/FormControlLabel';
 import WidgetPropTypes from '../../05_pages/NodeForm/WidgetPropTypes';
 
 const BooleanCheckbox = props => {
-  const { onChange, label, id, value, ...checkboxProps } = props;
+  const { onChange, label, value } = props;
+  const checkboxProps = Object.assign({}, props);
```

Comment:
What about doing something like, `const { entityTypeId, ...checkboxProps } = props;`?

Reply:
Yes, it's much better. :) Thanks